public/wp-content/themes/ColoredCow/functions.php

```diff
@@ -22,30 +22,27 @@ function cc_styles() {
 //add filter to remove margin above html
 add_filter('show_admin_bar','__return_false');
 
-function add_subscriber()
+
+function verify_field2()
 {
-    $post_title = $_POST['name'];
-    $post_phone = $_POST['phone'];
-    $post_email = $_POST['email'];
-    //$post_title = 'vaibhav';
-    //echo $_POST['title'];
-    $my_post = array(
-      'post_title'    => $post_title,
-      'post_phone'    => $post_phone,
-      'post_status'   => 'publish',
-      'post_type'     => 'guest'
-     );
-    
- 
-// Insert the post into the database
-    $post_id = wp_insert_post( $my_post );
-    if('$post_id')
+    //echo $_POST['value'];
+    global $wpdb;
+    $tablename = $wpdb->prefix."soiree";
+    $post_verify_email = sanitize_email($_POST['value']);
+    //echo $post_verify_email;
+    $datum = $wpdb->get_results("SELECT * FROM 23_sjpostmeta WHERE meta_value = '".$_POST['value']."'");
```

Comment:
remove hardcoded table name. This query won't run on a system where wpdb prefix is different.